CassandraScripts/casGenerateComments.js

```diff
@@ -14,6 +14,8 @@ const end   = new Date(2019, 11, 04);
 const range = moment.range(start, end)
 const million = 1000000
 var randomNormal = require('random-normal');
+const Uuid = require('cassandra-driver').types.Uuid;

```

Comment:
I'm curious why this is needed to generate a random ID? Wouldn't a random number from faker suffice?